src/Compilation/Config.zig

```diff
@@ -64,6 +65,8 @@ san_cov_trace_pc_guard: bool,
 
 pub const CFrontend = enum { clang, aro };
 
+pub const LTOMode = enum { default, auto, full, thin };
```

Comment:
```suggestion
pub const LtoMode = enum { none, full, thin };
```

* `LTO` -> `Lto` to follow Zig style.
* [Clang docs](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-flto) say that `auto` and `full` are equivalent.
* `default` should not exist at this level; a `Compilation.Config` should be fully resolved.